Source/WebCore/html/HTMLElement.h

```diff
@@ -46,9 +46,11 @@ struct TextRecognitionResult;
 
 enum class EnterKeyHint : uint8_t;
 enum class PageIsEditable : bool;
+enum class PopoverVisibilityState : bool;
+enum class ToggleState : bool;
+
 enum class FireEvents : bool { No, Yes };
```

Comment:
`SelectionRenderingBehavior` should come before this. All forward declarations before any non-forward declarations.